src/DemoUI/src/Controllers/OrderStateMappingController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Packlink\DemoUI\Controllers;
+
+use Packlink\DemoUI\Controllers\Models\Request;
+
+/**
+ * Class OrderStateMappingController
+ * @package Packlink\DemoUI\Controllers
+ */
+class OrderStateMappingController
+{
+    /**
+     * @param \Packlink\DemoUI\Controllers\Models\Request $request
+     */
+    public function getMappings(Request $request)
```

Comment:
Codacy found an issue: [Avoid unused parameters such as '$request'.](https://app.codacy.com/gh/packlink-dev/ecommerce_module_core/pullRequest?prid=5826400)

Reply:
This method is an extension point and the parameter is needed in the derived classes.